composer.json

```diff
@@ -38,6 +38,7 @@
         "mockery/mockery": "^1.0",
         "orchestra/database": "3.5.*|3.6.*|3.7.*|3.8.x-dev",
         "orchestra/testbench": "3.5.*|3.6.*|3.7.*|3.8.*",
+        "phpbench/phpbench": "@dev",
```

Comment:
Why do we require dev of phpbench?

Reply:
Because that is what they recommend https://phpbench.readthedocs.io/en/latest/installing.html#composer-install